SeaPublicWebsite/Controllers/CookieController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using GovUkDesignSystem.GovUkDesignSystemComponents;
+using Microsoft.AspNetCore.Mvc;
+using SeaPublicWebsite.Models.Cookies;
+using SeaPublicWebsite.Services.Cookies;
+
+namespace SeaPublicWebsite.Controllers;
+
+public class CookieController: Controller
+{
+    private readonly CookieService cookieService;
+
+    public CookieController(CookieService cookieService)
+    {
+        this.cookieService = cookieService;
+    }
+
+    [HttpGet("cookies")]
+    public IActionResult CookieSettings_Get(bool changesHaveBeenSaved = false)
+    {
+        cookieService.TryGetCookie<CookieSettings>(Request, cookieService.Configuration.CookieSettingsCookieName, out var cookie);
+        
+        var viewModel = new CookieSettingsViewModel
+        {
+            GoogleAnalytics = cookie?.GoogleAnalytics is true,
+            ChangesHaveBeenSaved = changesHaveBeenSaved
+        };
+        return View("CookieSettings", viewModel);
+    }
+
+    [HttpPost("cookies")]
+    [ValidateAntiForgeryToken]
```

Comment:
Instead of this, could you please add `options.Filters.Add(new AutoValidateAntiforgeryTokenAttribute());` to `services.AddControllersWithViews(options =>` in `Startup.cs`
That way all forms will be automatically validated